app/controllers/annotations_controller.rb

```diff
@@ -74,15 +75,16 @@ def admin_update
     @section = @question.section
     @phase = @section.phase
     if ex_save && guid_save
-      redirect_to admin_show_phase_path(id: @phase.id, section_id: @section.id, question_id: @question.id, edit: 'true'), notice: _('Information was successfully updated.')
+      typ = (!ex_save && !guid_save ? 'example answer and guidance' : (!guid_save ? 'guidance' : 'example answer'))
```

Comment:
because of the above if statement will requires both ex_save and guid_save to be true, this code will always return 'example answer'.  This should be revised with `example_answer.present?` and `guidance.present?`